uber/templates/preregistration/stripeForm.html

```diff
@@ -5,7 +5,8 @@
         data-key="{{ c.STRIPE_PUBLIC_KEY }}"
         {% if email %}data-email="{{ email }}"{% endif %}
         data-zip-code="true"
```

Comment:
I think this actually just disables the "remember me" checkbox, which is a different problem than we were having with the browser actually using autofill to pre-fill-in the CC details. It's still good to disable it, though.